plugins/linear-genome-view/src/LinearGenomeView/components/ImportForm.tsx

```diff
@@ -60,112 +68,160 @@ const ImportForm = observer(({ model }: { model: LGV }) => {
     }
   }, [assemblyManager, assemblyName])
 
-  function handleSelectedRegion(input: string) {
+  function setSelectedValue(selectedOption: BaseResult) {
+    let newValue = selectedOption.getLabel()
+    if (selectedOption instanceof RefSequenceResult) {
```

Comment:
why does there need to be instanceof checks here. both options call getLocation. is there a case where it is only uses getLabel instead of getLocation?

Reply:
my general concern, sort of mentioned last week, is that instanceof checks are kind of picky for code, especially for something pretty lightweight like a search result which I think should be a small blob of data. but, I am also curious why it needs getLocation in some cases, and only getLabel in others

Reply:
@cmdcolin yeah there are other parts of the code that only call the getLabel() for rendering the options in the dropdown. You are right, this was probably left behind on one of my refactors to the BaseResult class when only the reference sequence and locString result had a getLocation() method. This can be modified and that can all be simplified to just `setSelectedRegion(selectedOption.getLocation())` and there will be no need for `instanceof` checks

Reply:
nice :)) glad that works out